src/pages/patterns/notification-pattern/index.mdx

```diff
@@ -270,14 +254,49 @@ modal.
 </Column>
 </Row>
 
+### Actionable
+
+[Actionable](https://www.carbondesignsystem.com/components/notification/usage/)
+notifications allow for interactive elements within a notification styled like
+an inline or toast notification. Actionable notifications, since they require
+user interaction, take focus when triggered and can be highly disruptive to
+screen readers and keyboard users.
+
+Actionable notifications styled as inline notifications contain only one ghost
+button. A small “x” in the top right corner is used to dismiss inline
+notifications. Including the close button is optional and should not be included
+if it is critical for a user to read or interact with the notification.
+
+Actionable notifications styled as inline notifications can only one action
+button. If the toast includes an action button, then the notification should
+remain on screen until the user dismisses it. With the notification remaining
+open, the user has enough time to interact with the button without the toast
+closing too soon.
+
+#### Best practices:
+
+- Actionable notification should persist until user dismisses to give users time
+  in interact with the notification.
+- Only one action per notification.
+- Limit action labels to two words or less.
+
+<Row>
+<Column colLg={8}>
+
+![Banner notification in a product](/images/8_actionable_toast.png)
```

Comment:
The description should reflect the image you are showing. The image should also have the tertiary button with 16px after the text.